source/draw/gpu/GpuCanvas.ooc

```diff
@@ -40,21 +40,41 @@ GpuCanvasYuv420Semiplanar: class extends GpuSurface {
 		}
 	}
 
+	pen: Pen {
+		get { this _pen }
+		set(value) {
+			this _pen = value
+			yuv := value color toYuv()
+			this _target y canvas pen = Pen new(ColorBgra new(yuv y, 0, 0, 255), this pen width)
+			this _target uv canvas pen = Pen new(ColorBgra new(yuv u, yuv v, 0, 255), this pen width)
+		}
+	}
+
 	init: func (=_target, context: GpuContext) {
 		super(this _target size, context, context defaultMap, IntTransform2D identity)
-		this _target uv canvas clearColor = ColorBgra new(128, 128, 128, 128)
+		this _target uv canvas pen = Pen new(ColorBgra new(128, 128, 128, 128))
```

Comment:
I know it was already like this, but, won't this set the canvas to 50% transparency?